src/api/event/content-types/event/schema.json

```diff
@@ -73,6 +73,20 @@
     },
     "destaque": {
       "type": "boolean"
+    },
+    "subeventos": {
+      "type": "relation",
+      "relation": "oneToMany",
+      "target": "api::event.event"
+    },
+    "data_fim": {
```

Comment:
Tem algum motivo específico para usarmos `snake_case` ao invés de `camelCase`? Esses campos são colunas no banco de dados? Se não houver nenhuma restrição, acho que aderir ao `camelCase` aqui poderia ser uma boa, já que consumimos esses dados diretamente no front-end.

Reply:
A menos que isso se torne uma coluna no banco de dados, pois o Postgres não lida muito bem com `camelCase`, aí se esse for o caso, é mais fácil deixar com `snake_case` mesmo.

Reply:
Neste caso, casualmente, usamos snake_case para seguir a convenção do projeto.
Mas pelo que encontrei a partir da versão 4 do Strapi, o padrão é snake_case. (https://docs.strapi.io/dev-docs/migration/v3-to-v4/data/sql#column-name-casing)
Estes campos vão se tornar colunas no banco de dados sim.

Reply:
Seria legal depois ver para padronizar isso então, tornar uma regra pro cms.